yt_dlp/postprocessor/ffmpeg.py

```diff
@@ -754,10 +758,11 @@ def add(meta_list, info_list=None):
         add(('description', 'synopsis'), 'description')
         add(('purl', 'comment'), 'webpage_url')
         add('track', 'track_number')
-        add('artist', ('artist', 'creator', 'uploader', 'uploader_id'))
-        add('genre')
+        add('artist', ('artist_list', 'creator', 'uploader', 'uploader_id'))
+        add('composer', 'composer_list')
+        add('genre', 'genre_list')
         add('album')
-        add('album_artist')
+        add('album_artist', 'album_artist_list')
```

Comment:
These changes should reflect in https://github.com/yt-dlp/yt-dlp#modifying-metadata